drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -531,6 +538,7 @@ function DragAndDropTemplates(configuration) {
                     problemHeader,
                     h('p', {innerHTML: ctx.problem_html}),
                 ]),
+                h('div', {'className': 'sr'}, [forwardKeyboardHelpButtonTemplate(ctx)]),
```

Comment:
According to [Mark's comment on TNL-6022](https://openedx.atlassian.net/browse/TNL-6022?focusedCommentId=231733&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-231733), wrapping focusable content in `sr` only tags is a bad idea.
Can we go with the [design proposed by Christine](https://openedx.atlassian.net/browse/TNL-6022?focusedCommentId=231757&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-231757) instead?